apps/explorer/src/app/components/header/header.tsx

```diff
@@ -26,7 +26,11 @@ export const Header = ({
           {t('Vega Explorer')}
         </h1>
       </Link>
-      <button className="md:hidden" onClick={() => setMenuOpen(!menuOpen)}>
+      <button
+        data-testid="open-menu"
```

Comment:
if this is mobile , should be something more descriptive i.e open-burger-menu-btn or open-menu-mobile-btn

Reply:
As far as I know it's a standard React component which _could_ be used elsewhere which is why I've kept it consistent with the component name. I have updated the name of the variable that consumes it to openMobileMenuBtn.